addons/website_slides/models/slide_slide.py

```diff
@@ -108,7 +108,7 @@ class Slide(models.Model):
     }
     _order = 'sequence asc, is_category asc, id asc'
 
-    YOUTUBE_VIDEO_ID_REGEX = r'^.*(youtu.be\/|v\/|u\/\w\/|embed\/|watch\?v=|&v=)([^#&?]*).*'
+    YOUTUBE_VIDEO_ID_REGEX = r'^(?:(?:https?:)?//)?(?:www\.|m\.)?(?:youtube(-nocookie)?\.com/(?:embed/|v/|shorts/|live/|watch\?v=|watch\?.+&v=)|youtu\.be/)((?:\w|-){11})\S*$'
```

Comment:
In the regular expression, I'm not sure why we are capturing `-nocookie`. Is it necessary somewhere?

It would be great to reuse the same regex than `web_editor`/`website`/etc: Extracting a video id from a Youtube link is a common problem we have in several modules. It would be cool to have something consistent across all modules.